packages/drupal-admin-ui/src/components/02_atoms/Widgets/BooleanCheckbox.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import CheckBox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import { css } from 'emotion';
-import WidgetPropTypes from '../../../drupal-admin-ui/src/components/05_pages/NodeForm/WidgetPropTypes';
```

Comment:
I'm wondering whether it's a valid usecase to try to reuse these widgets?

Reply:
It seems like these widgets are more about functionality than the visual looks than the rest of the components so we probably should look at them separately at some point.